manticore/core/smtlib/solver.py

```diff
@@ -791,19 +792,18 @@ class SelectedSolver:
     def instance(cls):
         if consts.solver == consts.solver.auto:
             if cls.choice is None:
-                if os.path.exists(consts.yices_bin):
+                if shutil.which(consts.yices_bin):
```

Comment:
I think we should still support passing an exact path
```suggestion
                if os.path.exists(consts.yices_bin) or shutil.which(consts.yices_bin):
```

Reply:
Nevermind!!